packages/library/src/commonMain/kotlin/io/realm/RealmConfiguration.kt

```diff
@@ -33,6 +33,7 @@ import kotlin.reflect.KClass
 /**
  * Configuration for log events created by a Realm instance.
  */
+// FIXME Any reason for this to be public?
```

Comment:
It is exposed through `RealmConfiguration.log` so users can inspect the log configurationtion